.github/workflows/arm64_macos_bazel.yml

```diff
@@ -0,0 +1,36 @@
+# ref: https://github.com/actions/runner-images
+name: Arm64 MacOS Bazel
+
+on:
+  push:
+  pull_request:
+  schedule:
+    # min hours day(month) month day(week)
+    - cron: '0 0 7,22 * *'
+
+jobs:
+  # Building using the github runner environement directly.
+  bazel:
+    runs-on: macos-latest
```

Comment:
Can you add `# Using M1 processors, ref: https://github.com/actions/runner-images`

Reply:
done